plugin.video.repubblicaitaliana/resources/lib/plugin.py

```diff
@@ -46,22 +46,31 @@ def programmi_camera():
     liStyle.setProperty('isPlayable', 'true')
     xbmcplugin.addDirectoryItem(handle=G.PLUGIN_HANDLE, url=link, listitem=liStyle, isFolder=False)
 
+    titolo = 'Camera - Canale con sottotitoli automatici'
+    liStyle = xbmcgui.ListItem(titolo, offscreen=True)
+    link = 'plugin://plugin.video.tubed/?mode=play&video_id=4ZJz6alUDfc'
```

Comment:
would be nice if you provide an option for using youtube instead of tubed (i.e. let the user choose)

Reply:
Sometime ago someone tell me to remove youtube for tubed 😅
Why youtube is better ?